### ISSUE TITLE
Example project is too complex

### ISSUE THREAD

**Issue by naftis** [anncwb/vite-plugin-theme#12]
How should I use this in a smaller project? The example project is huge; it splits the plugin loading in vite.config.ts in so many different ways, that it's hard to follow how this plugin should be configured with for example 2 themes.

In other hand, the README is too simple. How should I create a simple 2 themed application and use the variables in my code?

**Comment by 0biWanKenobi** [comment]
Agreed, we need a simple example that shows how to switch between two themes.

**Comment by GitOfZGT** [comment]
look this plugin [@zougt/vite-plugin-theme-preprocessor](https://github.com/GitOfZGT/vite-plugin-theme-preprocessor)

**Comment by GitOfZGT** [comment]
look this plugin [@zougt/vite-plugin-theme-preprocessor](https://github.com/GitOfZGT/vite-plugin-theme-preprocessor)

**Comment by rj76** [comment]
It's not even configured in the example project. Meh.
